05_classify_mutations_multimodal/run_mutation_classification.py

```diff
@@ -65,6 +65,11 @@ def process_args():
                            'uncompressed data for all data types')
     opts.add_argument('--num_folds', type=int, default=4,
                       help='number of folds of cross-validation to run')
+    opts.add_argument('--overlap_data_types', nargs='*',
+                      default=['expression'],
+                      help='data types to define set of samples to use; e.g. '
+                           'set of data types for a model comparison, use only '
+                           'overlapping samples from these data types')
```

Comment:
I definitely think this is a good idea, but it will require a bit of thought/testing which I think would fit best in a separate PR - created #46 to do this in the future.